app/workers/email/enqueue_digest_worker.rb

```diff
@@ -0,0 +1,11 @@
+module Email
```

Comment:
as most of the workers follow the pattern of `"#{plural resource}::worker name" ` maybe we can rename this as `Emails::`